src/main/java/org/folio/rest/impl/UrlPath.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.rest.impl;
+
+public enum UrlPath {
```

Comment:
Are there any advantages of using an enum as opposed to a regular class with a set of constant strings?

```
public class UrlPath {
  public static final String CIRCULATION_REQUESTS_ALLOWED_SERVICE_POINTS = "/circulation/requests/allowed-service-points";
  ....
  }
```

Reply:
replaced with constants class, but seems like Sonar don't like it and suggest to create a getters for Url properties to avoid "hard coded using"